lib/galaxy/config.py

```diff
@@ -383,6 +390,14 @@ def __init__( self, **kwargs ):
         self.galaxy_infrastructure_url = galaxy_infrastructure_url
         self.galaxy_infrastructure_url_set = galaxy_infrastructure_url_set
 
+        # Verify host:port is available before continuing
+        try:
+            q = bind(*re.sub(r'http(s?)\://', '', self.galaxy_infrastructure_url).split(':'))
```

Comment:
In fact if you look at the end of the stack trace, it looks like Paste is trying to shutdown things properly. It would probably give you a better error if there wasn't a bug.

```
  File "/usr/lib64/python2.7/SocketServer.py", line 423, in __init__
    self.server_close()
  File "/home/xsebi/programs/galaxy/galaxy/eggs/Paste-1.7.5.1-py2.7.egg/paste/httpserver.py", line 1105, in server_close
    self.thread_pool.shutdown(60)
AttributeError: 'WSGIThreadPoolServer' object has no attribute 'thread_pool'
```

Just check for ``hasattr`` of ``thread_pool`` before accessing it where ever this is. Probably with that check in place paste would give a fine error?